### ISSUE TITLE
Doesnt work

### ISSUE THREAD

**Issue by ilovebashbunny** [hak5/bashbunny-payloads#359]
i placed lazagne.exe in the tools folder, and payload.txt, payload.ps1, lazagne.exe and lazagne-2.4 in the switch 2. im not sure but is this right?
after i "attacked" my computer with switch 2, lazagne .exe in the tools folder just disappears, probably it got deleted and idk why, and in the loot folder, the folder passwordgrabber is there but theres nothing in there. Pls help i need it

**Comment by Dragon8oy** [comment]
I feel bad for the devs in this repo, after they put a video on The Modern Rogue this place is going to be filled with people saying it isn't working and asking for help with little to no leads or logs.

**Comment by ilovebashbunny** [comment]
what? i didnt saw a video or anything
im just interested in the bash bunny so i bought it. i didnt came from youtube or any video
so can you pls give me an answe?

**Comment by hahnstep** [comment]
Maybe the antivirus from host delete the lazagne.exe ? Give a look in the logs windows defender or so.

**Comment by 0rion5** [comment]
Ilovebashbunny, I did the same thing, had the same result. The windows defender ate up the lasagne.exe. So I placed the file in the tools folder on the udisk with defender disabled from registry. That seemed to work.

**Comment by 0rion5** [comment]
Sorry for the late reply. I fixed it

**Comment by ilovebashbunny** [comment]
Okay, my keyboard wasnt US keyboard layout, thats why it didnt work.
I created now a payload who changed the keyboard layout to us (with powershell) and then it just runs the payload

**Comment by Jd305** [comment]
Explain what a bashbunny is?

**Comment by Treelovah** [comment]
@jd305 are you serious? 

Go to their website and read man... smh  🤦🏻‍♂️

**Comment by Treelovah** [comment]
As to the deleted lasagna.exe. That is your anti virus deleting your payloads. To get around this and continue testing, you can simply add lasagne.exe to the “allowed files” I Windows defender, you can actually have the bunny do this itself before executing the actual payload! :) I’ll post mine later. (..Mmm coffee)

**Comment by ilovebashbunny** [comment]
As I wrote before, my keyboard layout was false. And DUCKY_LANG didn't work because I live in Swiss and there is only a german DUCKY_LANG and the German and Swiss keyboard are different. So I wrote a code with Q ALTCODE which change the keyboard layout to en-US (via PowerShell). Then everything worked.
I also wrote a code which disables Windows Defender. Idk if the code still works when the Antivirus would be Avast or something.....
The Issue is solved
